### PR TITLE
Feature/origin oredr

### PR DESCRIPTION
**Pull request by JenniferShola**
@vietjew 

@MisterDeejay

### CODE REVIEW

File: lib/ingram_micro.rb [hunk 1]
```diff
@@ -15,4 +16,8 @@ def self.configure
     yield configuration
     configuration.assert_valid
   end
+
+  def self.generate_order_number(prefix='')
```
Comment (dl103) [inline lib/ingram_micro.rb]: isn't the order number supposed to be an integer? i'm wondering if allowing people to pass in a prefix would make people inclined to put in a string

Comment (sent-hil) [inline lib/ingram_micro.rb]: @dl103 is right, we'll need to enforce arg is a Integer. Some doc why it returns a string and why there's an random suffix added to the end would be appreciated 🍰

Comment (vietjew) [inline lib/ingram_micro.rb]: @dl103 @sent-hil that's a good point, we should enforce arg is an integer or string of an integer. We should also seek some clarity from IM about the requirements for that field.

To your second point @sent-hil, I agree that we need more/better documentation. I'll add comments, maybe utilizing TomDoc.

**Comment by sent-hil** [comment]
@vietjew sounds good. I'm going to merge this now, so we can send some test orders for IMM with this functionality.

**Pull request closed without merging**